COCO_DATA_FORMAT.md

```diff
@@ -94,3 +94,18 @@ Specifically, **only** image files are supported for the label files. The ground
     ]
 }
 ```
+
+## Regression
+
+Here is one example of the json file for regression task, where the "target" in the "annotations" field is a real-valued number (e.g. a score, an age, etc.). Note that each image should only have one regression target (i.e. there should be exactly one annotation for each image).
```

Comment:
Are we going to have multi-target regression datasets? If so, we could make it more general